### ISSUE TITLE
datetime insist on time

### ISSUE THREAD

**Issue by tobiasfabian** [getkirby/panel#668]
If the time field is empty the date won't be saved. It would be great if the time could be optional.

<img width="882" alt="bildschirmfoto 2015-11-19 um 18 34 55" src="https://cloud.githubusercontent.com/assets/1524319/11278737/f135c380-8eec-11e5-9e8b-18ff897ebf20.png">

```yaml
  startdate:
    label: Start Date
    type:  datetime

```

**Comment by bastianallgeier** [comment]
Hm, but what should be stored in such a case? A default time or no time at all? It would probably lead to situations where you expect the field to return a time, but you don't get anything usable back. Might be confusing.

**Comment by tobiasfabian** [comment]
I have a blueprint for events (some sort of calendar). Some events have a specific time. Some events take place the hole day and don’t have a specific time.

Maybe the developer should decide whether the time is required or not.
```yaml
startdate:
  label: Start Date
  type:  datetime
  time:
    required: true
```

**Comment by bastianallgeier** [comment]
ah, right, that sounds reasonable. I will keep this in mind for the next release.

**Comment by tobiasfabian** [comment]
n1ce.

**Comment by bastianallgeier** [comment]
This is finally implemented on the develop branch and will be in 2.4
